src/datatablerow/utils.jl

```diff
@@ -0,0 +1,187 @@
+# Rows grouping.
+# Maps row contents to the indices of all the equal rows.
+# Used by groupby(), join(), nonunique()
+immutable RowGroupDict{T<:AbstractDataTable}
+    dt::T                 # source data table
```

Comment:
@nalimilan do you have an example for how these comments can be written as docstrings?